app/Resources/views/base.html.twig

```diff
@@ -34,9 +45,10 @@
                 </div>
             </div>
         </header>
-        <div class="aui-page-panel">
+        <div class="aui-page-panel" style="min-height: 400px">
```

Comment:
Same here, goes here:
https://github.com/thecatontheflat/atlassian-vendor-tools/blob/master/web/assets/css/mpcrm.css

Reply:
Moved those inlines to mpcrm.css